README.org

```diff
@@ -1,17 +1,33 @@
+* Eventually Matrix
+
+  [[https://en.wikipedia.org/wiki/Eventual_consistency][Eventually]] Matrix is a
+  repository of knowledge about the Matrix ecosystem maintained by the community
+  for the community. It is the "bleeding edge" counterpart of the official
```

Comment:
Since "might look like in the future" convey the message that we could be going in the wrong direction, it would also have a bad effect on people wanting to support us, as they wouldn't want to go towards something that might simply not be the good road.

"Bleeding edge" was meant to convey a message of "most current knowledge repository".
What about:
> It is the compilation of latest and newest available information in terms of documentation, SDK and libraries.